SpywareFilter/sections/specific.txt

```diff
@@ -25,6 +25,7 @@
 !
 /analytics.js$domain=bataryapil.com|journey.com.tr|tozlu.com|elleshoes.com|freexcafe.com|rusvideos.tv|nakubani.ru|lun.com
 !
+||rake.11st.co.kr/log/
```

Comment:
I think `||rake.11st.co.kr^` would be ok.

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/81952/commits/d733d739f8fd64aca3a25bc40a688ad3d646974d